### PR TITLE
Trezor storage integration rebased

### PR DESCRIPTION
**Pull request by prusnak**
supersedes https://github.com/trezor/trezor-core/pull/467

### CODE REVIEW

File: tests/test_trezor.config.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def test_wipe(self):
         self.assertEqual(v1, None)
 
     def test_lock(self):
-        for _ in range(128):
+        for i in range(128):
```
Comment (prusnak) [inline tests/test_trezor.config.py]: i is not used, unnecessary change

Comment (andrewkozlik) [reply tests/test_trezor.config.py]: Agreed. I added that while debugging and forgot to clean up.

**Comment by prusnak** [comment]
Merged via 6d817a813d584a9a5240ad261e0cd64febd9eabf

Thanks!
